hazelcast/src/main/java/com/hazelcast/map/impl/proxy/MapProxyImpl.java

```diff
@@ -863,6 +868,7 @@ public <R> Map<K, R> executeOnEntries(@Nonnull EntryProcessor<K, V, R> entryProc
     @Override
     public <R> Map<K, R> executeOnEntries(@Nonnull EntryProcessor<K, V, R> entryProcessor,
                                           @Nonnull Predicate<K, V> predicate) {
+        checkDoesNotContainPagingPredicate(predicate, "executeOnEntries");
```

Comment:
minor: we can add null checks to here as well

Reply:
Don't know why but entry processors are not checked at all for being non-null in the MapProxyImpl file. I will only add for the predicate.